### PR TITLE
Evidence/ completion page updates

### PR DESCRIPTION
**Pull request by eadams17**
## WHAT
add updates to the completion page for Evidence (final round of UI updates)

## WHY
we want to show exemplars alongside student responses after completion of activities; there were also some requested design changes.

## HOW
include `first_strong_example` and `second_strong_example` properties in payload, refactor completion page to render exemplars and responses, add ability to add exemplars in internal tool; various styling updates

### Screenshots
Screen Shot 2022-02-17 at 5 26 30 PM

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)
https://www.notion.so/quill/Update-the-Completion-Page-efe12085e4c4470d921a5f1f095d6eed

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | yes

### CODE REVIEW

File: services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/explanationData.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const explanationData = {
     header: "This activity is not graded",
     imageData: {
       imageAlt: "An illustration of an A+ that is crossed out",
-      imageUrl: "images/comprehension/no-grades.svg"
+      imageUrl: "images/evidence/no-grades.svg"
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/explanationData.ts:7]: These all seem like renames.  Is it worth removing the old SVG files from the `quill-cdn` directory so we don't have them taking up space?

File: services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/postActivitySlide.tsx [hunk 2]
```diff
@@ -1,38 +1,85 @@
...
-    // we subtract 3 to account for a student having 3 optimal first attempts
-    const calculatedCount = count - 3;
-    if(calculatedCount === 1) {
-      return 'You made 1 revision!';
-    }
-    return `You made ${calculatedCount} revisions!`;
+import { BECAUSE, BUT, SO } from '../../../Shared/index'
+import { PromptInterface } from "../../../Staff/interfaces/evidenceInterfaces";
+
+const FIRST_STRONG_EXAMPLE = 'first_strong_example';
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/postActivitySlide.tsx]: We should probably use the defined constants import here as well as in our tests.

File: services/QuillLMS/client/app/bundles/Staff/components/evidence/configureSettings/promptsForm.tsx [hunk 3]
```diff
@@ -16,38 +16,84 @@ interface PromptsFormProps {
...
           value={activityButPrompt.text}
         />
+        <Input
+          className="but-example"
+          handleChange={handleSetButPromptFirstStrongExample}
+          label="But First Exemplar"
+          value={activityButPrompt.first_strong_example}
+        />
+        <Input
+          className="But-example"
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/Staff/components/evidence/configureSettings/promptsForm.tsx]: Not that it matters, but is the `But` here accidentally capitalized?

**Review by anathomical — APPROVED** [review]
A couple of minor notes and suggestions, but nothing that seems like it should prevent this from being merged.

File: services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/explanationData.ts (hunk 1 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/explanationData.ts:7]: yeah good call, I'll remove them!

File: services/QuillLMS/client/app/bundles/Staff/components/evidence/configureSettings/promptsForm.tsx (hunk 3 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Staff/components/evidence/configureSettings/promptsForm.tsx]: it is, I'll change it

File: services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/postActivitySlide.tsx (hunk 2 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/postActivitySlide.tsx]: good catch!

**Review by emilia-friedberg — APPROVED** [review]
LGTM!

**Pull request merged**